### PR TITLE
Added Cloning context

### LINKED ISSUE #4851: Provide context on cloning for new users
### What article on docs.github.com is affected?

"[Create a repo](https://docs.github.com/en/github/getting-started-with-github/create-a-repo)"

### What part(s) of the article would you like to see updated?

We got feedback in our helpfulness survey that this article should explain what it means to clone a repo to one's computer. While the article doesn't focus on what happens after creating a repo on GitHub, we can include a link to our "[About repositories](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-repositories)" article to give more context, and to "[Cloning a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)" for next steps.

### Additional information

This issue is for a special project and is not currently available for contributors to work on.

### PR DESCRIPTION
**Pull request by ShreyaPrasad1209**
### Why:

Closes #4851 

### What's being changed: 
- Added a sentence under Create a Repo headline which links to [About repositories](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-repositories). 
- Added link to [Clone a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) under Celebrate header

image

image

### Check off the following:

- [x] I have reviewed my changes in staging (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/CONTRIBUTING.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
@ShreyaPrasad1209 Thanks for opening a PR 🙌 I'll triage this for review ⚡

File: content/github/getting-started-with-github/create-a-repo.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ topics:
 
 {% if currentVersion == "free-pro-team@latest" %}
 
-You can store a variety of projects in {% data variables.product.product_name %} repositories, including open source projects. With [open source projects](http://opensource.org/about), you can share code to make better, more reliable software.
+You can store a variety of projects in {% data variables.product.product_name %} repositories, including open source projects. With [open source projects](http://opensource.org/about), you can share code to make better, more reliable software. Also, there are various types of repositories for different purposes about which you can read more on [About Repositories](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-repositories).
```
Comment (lecoursen) [inline content/github/getting-started-with-github/create-a-repo.md:20]: "Various types of repositories" and "different purposes" is a bit vague. Let's be a bit more specific (but still concise!) about the information folks will find in the link.

We should also use relative links, formatted like this.

```suggestion
You can store a variety of projects in {% data variables.product.product_name %} repositories, including open source projects. With [open source projects](http://opensource.org/about), you can share code to make better, more reliable software. You can use repositories to collaborate with others and track your work. For more information, see "[About repositories](/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-repositories)."
```

File: content/github/getting-started-with-github/create-a-repo.md [hunk 2]
```diff
@@ -80,6 +80,7 @@ Congratulations! You have now created a repository, including a *README* file, a
 
 - "[Set up Git](/articles/set-up-git)"
 - **Create a repository**
+- "[Clone a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)"
```
Comment (lecoursen) [inline content/github/getting-started-with-github/create-a-repo.md:83]:
```suggestion
- "[Clone a repository](/github/creating-cloning-and-archiving-repositories/cloning-a-repository)"
```

**Review by lecoursen — DISMISSED** [review]
Thanks @ShreyaPrasad1209! I left a few small suggestions, which I'll apply now and then merge! 🎉

**Comment by lecoursen** [comment]
Whoops, sorry... I spoke too soon! 🙈 @ShreyaPrasad1209, you have a merge conflict due to a recent site reorganization. I tried to fix it for you locally but am unable to do so because I don't have access to your branch. Could you please add me as a collaborator to your fork? That way I can fix the conflict there!

https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository

**Comment by ShreyaPrasad1209** [comment]
@lecoursen I have added you as a collaborator! 🚀 
Quick question - I noticed that GitHub Docs now requires node version 16 for successful local setup. So I upgraded and pushed all of the files to my remote repository and now the Node.js Tests - Windows workflow run has failed. I'm attaching the screenshot.

image

Note: This error is independent of this PR, I am asking for recent commits I made to my remote GitHub Docs repository.

**Comment by lecoursen** [comment]
> I think that I should have not pushed my updated package.json because it might be interfering with the original codebase's package.json file. Can you please confirm whether my assumption is correct?

This won't always cause problems but it does occasionally... so just to be safe I refrain from committing my package.json changes. Thanks for asking!

Looks like you resolved the conflicts yourself... thanks! ⭐ I'll go ahead and merge now. 😎

**Comment by ShreyaPrasad1209** [comment]
Thank you for the confirmation!😊

**Review by lecoursen — APPROVED** [review]

**Comment by janiceilene** [comment]
I'm going to try closing this and reopening to rerun the staging deployment!

**Pull request merged**